ga4gh/datamodel/variants.py

```diff
@@ -423,6 +440,11 @@ def getVariants(self, referenceName, startPosition, endPosition,
         # protocol version 0.6
         if callSetIds is None:
             callSetIds = []
+        else:
+            for call in callSetIds:
+                if call not in self._callSetIds:
+                    raise exceptions.CallSetNotInVariantSetException(
```

Comment:
Have we defined this exception? Also, I don't think we need to pass the dict of callSetIds to the exception. I would use 
```python
raise exceptions.CallSetNotInVariantSetException(self, callSetId)
```
The exception message could then be "CallSet '{}' not in VariantSet '{}'".format(callSetId, variantSet.getId())